docs/what.md

```diff
@@ -0,0 +1,16 @@
+# All about controllers
+
+If you've come here from the [README](../README.md), you already know that this repo holds controllers. But what even _is_ a controller?
+
+The short answer is that it's our solution to managing and disseminating state changes within our products.
+
+Every application needs to manage state. Some state is temporary. Perhaps the
```

Comment:
Hmm, this paragraph seems a bit... meandering. I like the idea of establishing what application state is; just saying "state" can be a bit too abstract to grasp. Maybe we can reformat this to separate the examples of application state from the details about some state being transient, some persisted, etc.